### ISSUE TITLE
Can't toggle comments in SQL documents

### ISSUE THREAD

**Issue by edelsonc** [adobe/brackets#12577]
Hello

Comment toggling doesn't work on OS X in .sql scripts.

I'm using Brackets Release 1.7.

Thanks

**Comment by ficristo** [comment]
Seems a general problem: tryed with sql, cc, and js snippet of code.
```
/*
 *  Comment adjacent to the gutter cannot be folded.
 */
 /*
 *  Comment with one space between the gutter can be folded.
 */
```
/cc @thehogfather

**Comment by thehogfather** [comment]
@edelsonc  @ficristo This is a bug from the version of CodeMirror included in Brackets. The issue will disappear once brackets upgrades code mirror whenever that is.

As a workaround, the comments are foldable if there is a space before the first `/*`.

**Comment by ficristo** [comment]
@thehogfather good to know!

Another thing is that would be nice to be able to collapse comments that span on multiple lines (and use line comments).
```sql
-- A comment
-- that span
-- on multiple lines
```
Maybe too hard / peformance sensitive but still...

@edelsonc I'm sorry to have hijacket your issue...

**Comment by thehogfather** [comment]
@ficristo would require writing a range finder for single line comments that span multiple consecutive lines - so not too hard. 

Might be worth creating a separate issue so we can discuss the implications of such a use case. The actual range finder should be relatively easy to write.

**Comment by ficristo** [comment]
We [update CodeMirror](https://github.com/adobe/brackets/pull/12613) and @thehogfather fixed a [problem](https://github.com/adobe/brackets/pull/12521) with CodeFolding, both of which should be available on 1.8.
Closing since it is fixed.
